src/app/Components/MovieList/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, connect } from 'react-redux';
+
+import { getMovies } from './duck/operations';
+import { SUCCESS } from '../../../common/constants';
+
+const renderMovies = (movies) => (
+  <div className="flex mt-8 ml-24 mr-24 flex-wrap">
+    {movies.map(({ poster_url, title }) => (
+      <div className="mr-1 mb-4" key={title}>
+        <img src={poster_url} alt={title} />
+      </div>
+    ))}
+  </div>
+);
+
+const renderTitle = (title) => (
+  <div className="">
+    <h1 className="ml-24 text-3xl font-semibold">{title}</h1>
+  </div>
+);
+
+const MovieList = (props) => {
```

Comment:
We can use destructuring for the props here, so we don't need to repeat `props.movieList`

Reply:
Also, it would be nice to add proptype definitions, in order to be consistent across all components.